svgcheck/test.py

```diff
@@ -196,42 +191,18 @@ def test_svg_ViewboxBoth(self):
         test_svg_file(self, "viewBox-both.svg")
 
 
-def test_rfc_file(tester, fileName):
```

Comment:
Earlier this test was only run on Windows OS.
With the new changes, this test doesn't even make sense, because it's only calling an xml2rfc class.